Packages/packages-lock.json

```diff
@@ -1,13 +1,15 @@
 {
   "dependencies": {
     "com.unity.2d.animation": {
-      "version": "3.2.2",
+      "version": "5.0.9",
```

Comment:
Do you really want to increase unity version?

Reply:
Changed back to the previous version